src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitRepository.kt

```diff
@@ -83,6 +83,14 @@ interface VisitRepository : JpaRepository<Visit, Long>, JpaSpecificationExecutor
     endDateTime: LocalDateTime,
   ): Boolean
 
+  @Query(
+    "SELECT count(v) > 0 FROM Visit v " +
+      "WHERE v.sessionTemplateReference = :sessionTemplateReference",
```

Comment:
need to add status as BOOKED

Reply:
resolved after having a chat with @AledWEvans as this cannot be done once the session template goes Live and is used to create visits whatever the Status.